pkg/svcat/service-catalog/broker.go

```diff
@@ -109,8 +112,49 @@ func (sdk *SDK) RetrieveBroker(name string) (*v1beta1.ClusterServiceBroker, erro
 	return broker, nil
 }
 
-// RetrieveNamespacedBroker gets a broker by its name & namespace.
-func (sdk *SDK) RetrieveNamespacedBroker(namespace string, name string) (*v1beta1.ServiceBroker, error) {
+// RetrieveBrokerByID gets a broker by its k8s name
+func (sdk *SDK) RetrieveBrokerByID(kubeName string, opts ScopeOptions) (Broker, error) {
+	var csb *v1beta1.ClusterServiceBroker
+	var sb *v1beta1.ServiceBroker
+	var err error
+	if opts.Scope.Matches(ClusterScope) {
+		csb, err = sdk.ServiceCatalog().ClusterServiceBrokers().Get(kubeName, v1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			csb = nil
+		}
+		if err != nil && !apierrors.IsNotFound(err) {
+			return nil, fmt.Errorf("unable to search clusterservicebrokers by k8s name (%s)", err)
+		}
+	}
```

Comment:
return csb here?

Reply:
its possible we get both a sb and csb back, if they were created with the same name, which is possible as they're different types